2d/benchmarks/quiescent_water_probe_benchmark/redist_n.py

```diff
@@ -74,7 +74,7 @@
 multilevelNonlinearSolver  = NonlinearSolvers.Newton
 levelNonlinearSolver       = NonlinearSolvers.Newton
 
-nonlinearSmoother = NonlinearSolvers.GaussSeidel
+nonlinearSmoother = NonlinearSolvers.NLGaussSeidel
```

Comment:
@Pedrohrw, I guess this change give you better results, does it?

Reply:
@Giovanni-Cozzuto-1989, actually I didn't compare it with the previous one. I just updated it to the new code. Should I check with the previous one?